src/components/userinput.jsx

```diff
@@ -0,0 +1,9 @@
+const UserInputs = () => {
+  return (
+    <div class="user-input-display rounded-10 mt-1 flex h-[15vh] flex-col justify-center bg-themeOne-user-input-background ">
+      <div class="user-input w-97 h-[7vh] justify-end text-right text-[32px]">0</div>
+    </div>
+  );
```

Comment:
Is this input or output? Anyway, can we use `<Display />` instead for better naming.